src/App.js

```diff
@@ -9,22 +9,29 @@ import { Duvidas } from './views/Duvidas'
 import { Orgs } from './views/Organizacoes'
 
 import './styles/reset.css'
+import { Menu } from './components/Menu'
+import { ThemeProvider } from 'styled-components'
+import { themeLight } from './util/themes'
 
 function App() {
   return (
-    <div className="App">
-      <Switch>
-        <Route path="/sobre" component={Sobre} />
-        <Route path="/horas" component={Horas} />
-        <Route path="/atividades" component={Atividades} />
-        <Route path="/orgs" component={Orgs} />
-        <Route path="/duvidas" component={Duvidas} />
-        <Route path="/perfil" component={Perfil} />
-        <Route path="*">
-          <Redirect to="/sobre" />
-        </Route>
-      </Switch>
-    </div>
+    <ThemeProvider theme={themeLight}>
+      <div className="App">
+        <Menu>
+          <Switch>
+            <Route path="/sobre" component={Sobre} />
+            <Route path="/horas" component={Horas} />
+            <Route path="/atividades" component={Atividades} />
+            <Route path="/orgs" component={Orgs} />
+            <Route path="/duvidas" component={Duvidas} />
+            <Route path="/perfil" component={Perfil} />
+            <Route path="*">
```

Comment:
Tu já definiu todas essas rotas no arquivo de constantes. Com isso tu pode utilizar essa constante de `pages` para criar as rotas de forma automatica.

```suggestion
            { pages.map(p => <Route path={p.path} component={p.component} /> }
```